platform/darwin/src/MGLRendererConfiguration.mm

```diff
@@ -69,10 +69,38 @@ - (const float)scaleFactor {
     return mbgl::optional<std::string>();
 }
 
-- (std::string)localFontFamilyName {
-    NSString *fontFamilyName = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"MGLIdeographicFontFamilyName"];
+- (mbgl::optional<std::string>)localFontFamilyName {
+    
+    std::string systemFontFamilyName;
+#if TARGET_OS_IPHONE
+    systemFontFamilyName = std::string([[UIFont systemFontOfSize:0 weight:UIFontWeightRegular].familyName UTF8String]);
+#else
+    systemFontFamilyName = std::string([[NSFont systemFontOfSize:0 weight:NSFontWeightRegular].familyName UTF8String]);
+#endif
 
-    return fontFamilyName ? std::string([fontFamilyName UTF8String]) : std::string("PingFang");
+    id fontFamilyName = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"MGLIdeographicFontFamilyName"];
+    if([fontFamilyName isKindOfClass:[NSString class]])
+    {
+        if([fontFamilyName isEqualToString:@"NULL"]){
```

Comment:
Let's uppercase fontFamilyName, so that we also catch `null`, `Null`, etc...

Should we also accept `False`, `false`, `NO`, `no`, `0`, etc..?